packages/blockchain-link-types/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.0.18-beta.1
+
+-   refactor(suite): signed transaction data stored in redux send form state (2ea31ab5d6)
```

Comment:
this is an example of poorly structured commit. for the packages in `/packages` folder, or at least for those we publish to npm, we shoudl write commits more atomically. 

so this [change](https://github.com/trezor/trezor-suite/commit/2ea31ab5d65f04c33b5f1f28c3c4f007971dbff4#diff-0515aefbe868af7bb866d5064b95489b37e45e3fb6086a8c5e10f0fa58f87274R6) shoudl have been a single commit with a commit messages something like: chore(blockchain-link-types): export Transaction type"

Reply:
cc @PeKne